seminare/submits/utils.py

```diff
@@ -0,0 +1,81 @@
+from io import BytesIO
+from pathlib import Path
+
+from django.contrib.auth.models import User
+from django.core.files import File
+from django.core.files.base import ContentFile
+from judge_client.client import JudgeClient, Submit
+from PIL import Image
+from reportlab.lib.pagesizes import A4
+from reportlab.lib.utils import ImageReader
+from reportlab.pdfgen import canvas
+
+from seminare.settings import JUDGE_TOKEN, JUDGE_URL
```

Comment:
```suggestion
from django.conf import settings
```

https://docs.djangoproject.com/en/5.2/topics/settings/#using-settings-in-python-code